app/controllers/amusement_parks_controller.rb

```diff
@@ -0,0 +1,10 @@
+class AmusementParksController < ApplicationController
+
+  def show
+    @parks = AmusementPark.find(params[:id])
+
+    @mechanics_on_site = @parks.rides
```

Comment:
We want all of our database logic to live in the model. So you will want this to live in a method in your `amusement_park.rb` model class.